ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,108 @@
+    n) Sport mode deprecated
+    o) SURFTRAK_MODE allows disabling surface tracking in Loiter, AltHold, etc
+3) Custom build server support (see https://custom.ardupilot.org)
+4) Lua scripting improvements
+    a) ahrs::get_location replaces get_position (get_position still works for now)
+    b) Auto support for NAV_SCRIPT_TIME commands (Lua within Auto)
+    c) Frame string support (allows scripting based frame to display custom description after startup)
+    d) Parameter support (no need to always use SCR_USERx)
+    e) Script logged to onboard log (can be disabled by setting SCR_DEBUG_OPTS)
+5) New autopilots supported
+    a) AirLink
+    b) DevEBoxH7v2
+    c) JHEMCU GSF405A
+    d) KakuteH7
+    e) MambaF405US-I2C
+    f) MatekF405-TE
+    g) PixC4-Jetson
+    h) Pixhawk5X
+    i) QioTekZealotH743
+    j) RPI-Zero2W
+    k) SPRacingH7
+    l) Swan-K1
+6) Safety improvements
+    a) Dijkstra's avoidance performance improvements including converting to A*
+    b) Motor PWM range always uses MOT_PWM_MIN/MAX (if these are not set, they are defaulted to RC3_MIN/MAX values)
+    c) Parachute released arming check added
+    d) Pre-arm check of IMU heater temp
+    e) Pre-arm check of rangefinder health
+    f) Pre-arm check of throttle position skipped if PILOT_THR_BHV is "Feedback from mid stick"
+7) Sensor driver enhancements
+    a) ADIS16470 and ADIS16507 IMU support
+    b) Airspeed sensor support (reporting only, not used for estimation or control)
+    c) AK09918 compass support
+    d) Battery monitor supports voltage offset (see BATTx_VLT_OFFSET)
+    e) Benewake TFMiniPlus I2C address defaults correctly
+    f) Buzzer can be connected to any GPIO on any board
+    g) Compass calibration (in-flight) uses GSF for better accuracy
+    h) CSRF telemetry link reports link quality in RSSI
+    i) DroneCan (aka UAVCAN) battery monitors support scaling (see BATTx_CURR_MULT)
+    j) Electronic Fuel Injection support incl Lutan EFI
+    k) FETtecOneWire resyncs if EMI causes lost bytes
+    l) IMU heater params renamed to BRD_HEAT_xxx
+    m) Landing gear enable parameter added (see LGR_ENABLE)
+    n) Lightware SF40C ver 0.9 support removed (ver 1.0 and higher still supported)
+    o) Maxbotix serial sonar driver support RNGFNDx_SCALING parameter to support for varieties of sensor
+    p) MPPT solar charge controller support
+    q) MTK GPS driver removed
+    r) Optical flow in-flight calibration
+    s) Proximity sensor min and max range (see PRX_MIN, PRX_MAX)
+    t) uLanding (aka USD1) radar provides average of last few samples
+    u) Unicore NMEA GPS support for yaw and 3D velocity 
+8) TradHeli enhancements
+    a) Attitude control default gains improved
+        - ATC_RAT_RLL_FF, _I, _IMAX, _FLTE, _FLTT
+        - ATC_RAT_PIT_FF, _I, _IMAX, _FLTE, _FLTT
+        - ATC_RAT_YAW_IMAX, _FLTT
+    b) AutoTune mode
+    c) Collective setup improvements
+    d) _VFF params renamed to _FF
```

Comment:
```suggestion
    d) _VFF params renamed to _FF
    e) Improved Rotor Speed Controller Internal Governor (users required to retune governor)
    f) Cooldown timer in Rotor Speed Controller for ICE and turbine engines
```

Reply:
txs, added two new items (slightly reworded):
    d) Rotor Speed Controller Internal Governor improvements (users required to retune governor)
    e) Rotor Speed Controller Cooldown timer added for ICE and turbine engines